.vsts-pr.yml

```diff
@@ -0,0 +1,150 @@
+# Pipeline: https://dev.azure.com/dnceng-public/public/_build?definitionId=231
+
+variables:
+  - name: _TeamName
+    value: DotNetCore
+  - name: Build.Repository.Clean
+    value: true
+  - name: Codeql.Enabled
+    value: ${{ eq(variables['System.TeamProject'], 'internal') }}
+  - name: Codeql.TSAEnabled
+    value: ${{ eq(variables['System.TeamProject'], 'internal') }}
+  - ${{ if eq(variables['System.TeamProject'], 'internal') }}:
+    - group: DotNet-Sign-SDLValidation-Params
+
+# CI and PR triggers
+trigger:
+  batch: true
+  branches:
+    include:
+    - main
```

Comment:
In the past, I've been bitten by this if I am updating a servicing branch.  If this repo will use servicing branches, I recommend an entry for it like `rel/*` or `release-*`.  Obviously if this tool will never be servicing older versions, its not needed.

Reply:
Great feedback.  Currently, we don't have release branches.  We've only had prereleases.  We'll cross that bridge when we get there.